plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -161,7 +169,30 @@ export function handleInvalidQuery(
   return result;
 }
 
+export function handleExecutionResult(
+  config: PgInstrumentationConfig,
+  span: Span,
+  pgResult: pgTypes.QueryResult | pgTypes.QueryArrayResult | unknown
+) {
+  if (config.responseHook !== undefined && pgResult !== undefined) {
```

Comment:
Same comment from the other "add responseHook", why explicitly only checking against undefined? Why not ```if (config.responseHook && pg...)```

Or event better ```if (typeof config.responseHook === "function" && pg...)```

Reply:
I think we should just have `if (typeof config.responseHook === "function")` if user adds hook we should always run it whether the `pgResult` is undefined or not as this is also some kind of information for someone. Preventing running this hook in such case will confuse user why the hook didn't run soo I would definitely remove it.

Reply:
@obecny pushed 👍🏻  good idea!